### PR TITLE
Edited tutorial tool to embed images and added project submission 

### PR DESCRIPTION
**Pull request by caitlinoxleyhmb301**

### CODE REVIEW

File: tools/iqtree.md [hunk 1]
```diff
@@ -22,16 +22,16 @@ Using the SRA number as the main identifier of each obelisk sample sequence.
 eg."(OBELISK_SRA_NUMBER) >
 obelisk_sequence""
 Then I saved the file with a ".txt" suffix.
-![Alt text](/Users/caitlinoxley/Downloads/tutorial_q1.png)
+![Alt text](https://github.com/ababaian/VIRUSxDISCVRY/blob/main/tools/img/iqtree/tutorial_q1.png)
```
Comment (ababaian) [inline tools/iqtree.md]: Very close! Great job. You can actually simplify this to "point" from the place where this file lives. so change this to 
```
![Make this a better description](img/iqtree/tutorial_q1.png)
```

File: virus/img/Obelisk371/temp
Comment (ababaian) [inline virus/img/Obelisk371/temp:1]: please remove temporary files.

**Review by ababaian — CHANGES_REQUESTED** [review]

**Review by ababaian — APPROVED** [review]

**Comment by ababaian** [comment]
Looking good, just make sure you keep the main titles for a "paper"

**Comment by caitlinoxleyhmb301** [comment]
Ok, I think I made all the necessary changes.

**Pull request merged**